environment.yml

```diff
@@ -24,6 +24,6 @@ dependencies:
   - mock=4.0
   - recommonmark=0.7
   - sphinx=6.0
+  - sphinx-rtd-theme=3.0.1
   - pip:
      - graphspace_python==1.3.1
```

Comment:
I believe the sphinx dependencies came from [graphspace](https://github.com/adbharadwaj/graphspace-python/blob/master/requirements.txt). That is currently a required packaged for one of our pathway output file formats, so all of its dependencies may also be required.